cpp/src/aztec/honk/composer/composer_helper/composer_helper.cpp

```diff
@@ -218,6 +218,8 @@ std::shared_ptr<waffle::proving_key> ComposerHelper<CircuitConstructor>::compute
 
     // TODO(Cody): this is a workaround
     circuit_proving_key->polynomial_cache.put("z_perm_lagrange", Polynomial<barretenberg::fr>(1));
+    // used in test_verification_key_creation
+    circuit_proving_key->polynomial_cache.put("pow_zeta", Polynomial<barretenberg::fr>(1));
```

Comment:
Is this necessary? Same question for `z_perm_lagrange`

Reply:
I just tried removing running some tests. No, it seems they are not needed. I don't recall what was being worked around. Will remove.